eng/common/scripts/Package-Properties.ps1

```diff
@@ -154,7 +154,7 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
 
     if ($AdditionalValidationPackagesFromPackageSetFn -and (Test-Path "Function:$AdditionalValidationPackagesFromPackageSetFn"))
     {
-        $packagesWithChanges += &$AdditionalValidationPackagesFromPackageSetFn $packagesWithChanges $diff
+        $packagesWithChanges += &$AdditionalValidationPackagesFromPackageSetFn $packagesWithChanges $diff $allPackageProperties
```

Comment:
Arguments:

- Existing set of packages we consider to be changed
- The diff object
- The entire list of resolved package properties

This way, when we identify packages that we want to use, we can just find the package props for it and add it to the list.